### PR TITLE
Feature #248 - Implement review RUD view

### LINKED ISSUE #248: SP: 1 Implement RUD endpoint for reviews
Part of the User Story #41

### PR DESCRIPTION
**Pull request by OleksiiDatsiuk**
I have implemented an additional permission class to check, whether the user is an owner of a particular review.
Also implemented RUD view to retrieve/update/delete review.

### CODE REVIEW

File: beauty/api/views/review_views.py [hunk 1]
```diff
@@ -39,7 +41,32 @@ def get(self, request, to_user):
         return Response(serialized_data.data, status=status.HTTP_200_OK)
 
 
-class ReviewDisplayDetailView(RetrieveAPIView):
+class ReviewRUDView(RetrieveUpdateDestroyAPIView):
     """View for retrieving specific review."""
     queryset = Review.objects
     serializer_class = ReviewDisplaySerializer
+    permission_classes = (IsAdminOrCurrentReviewOwner, )
```
Comment (morento101) [inline beauty/api/views/review_views.py:48]: How does regular user will retrieve review, if only owner of the review can see it?

File: beauty/api/permissions.py [hunk 2]
```diff
@@ -105,3 +105,25 @@ def has_object_permission(self, request, view, obj):
...
+class IsAdminOrCurrentReviewOwner(permissions.IsAuthenticated):
+    """Object-level permission to allow only authors of review or admins to access it."""
+
+    def has_object_permission(self, request, view, obj):
+        """Verify that the current user is a review owner or an administrator."""
+        try:
+            has_access = request.user.is_admin or (obj.from_user == request.user)
+            if has_access:
+                logger.info(f"User {request.user.id} permission check. "
+                            f"Access granted",
```
Comment (pazuzu-ua) [inline beauty/api/permissions.py]: Why f string here?

File: beauty/api/views/review_views.py (hunk 1 above)
Comment (pazuzu-ua) [reply beauty/api/views/review_views.py:48]: Even Specialist can't see it...

Comment (OleksiiDatsiuk) [reply beauty/api/views/review_views.py:48]: Customer is not supposed to retrieve particular review, he can obtain list of all reviews and for that reasons I have created view earlier

File: beauty/api/permissions.py (hunk 2 above)
Comment (OleksiiDatsiuk) [reply beauty/api/permissions.py]: Got it

**Comment by OleksiiDatsiuk** [comment]
Regular user or a customer are not supposed to obtain or update specific review. They are only able to get list of all reviews. I am not sure if there is any reason to make this RUD's get method accessible to other users, because there is other view, whish purpose is to return list of all reviews of a particular user

**Review by pazuzu-ua — APPROVED** [review]

**Review by morento101 — APPROVED** [review]

**Review by datsiuk-ira — APPROVED** [review]

**Pull request merged**